src/services/car/inventory.ts

```diff
@@ -41,6 +42,25 @@ export const fetchListing = async ({
   return sanitizeListing(listing)
 }
 
+export const bulkFetchListing = async ({
+  listingIds,
+  options = {},
+}: {
+  listingIds: number[]
+  options?: ApiCallOptions
+}): Promise<BulkFetchResponse<Listing>[]> => {
+  const listings = await postData({
+    path: "listings/bulk-get",
+    body: { elements: listingIds },
+    options,
+  })
+
+  return listings.map((listing) => ({
+    id: listing.id,
+    payload: sanitizeListing(listing.payload),
```

Comment:
There seems to be some specific logic regarding date handling not quite sure what the intention behind feels a bit as it would be for a specific use case at first glance...
I would either drop it or add a test to make sure it's returned as expected

Reply:
the method should return the same as `fetchListing` but in bulk and not just a single one. That's why I added it. `sanitizeListing` does some magic with the dates.

Reply:
added a test for it